vochain/types.go

```diff
@@ -59,13 +64,73 @@ type QueryData struct {
 	ProcessType string `json:"type,omitempty"`
 }
 
+// ________________________ TRANSACTION COSTS __________________________
+// TransactionCosts describes how much each operation should cost
+type TransactionCosts struct {
+	SetProcess            uint64 `json:"Tx_SetProcess" state:"c_setProcess"`
+	RegisterKey           uint64 `json:"Tx_RegisterKey" state:"c_registerKey"`
+	NewProcess            uint64 `json:"Tx_NewProcess" state:"c_newProcess"`
+	SendTokens            uint64 `json:"Tx_SendTokens" state:"c_sendTokens"`
+	SetAccountInfo        uint64 `json:"Tx_SetAccountInfo" state:"c_setAccountInfo"`
+	AddDelegateForAccount uint64 `json:"Tx_AddDelegateForAccount" state:"c_addDelegateForAccount"`
+	DelDelegateForAccount uint64 `json:"Tx_DelDelegateForAccount" state:"c_delDelegateForAccount"`
+	CollectFaucet         uint64 `json:"Tx_CollectFaucet" state:"c_collectFaucet"`
+}
+
+// StructAsBytes returns the contents of TransactionCosts as a map. Its purpose
+// is to keep knowledge of TransactionCosts' fields within itself, so the
+// function using it only needs to iterate over the key-values.
+func (t *TransactionCosts) StructAsBytes() (b map[string][]byte, err error) {
```

Comment:
nit: avoid named returns in general. see https://github.com/golang/go/wiki/CodeReviewComments#naked-returns and https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters